.env

```diff
@@ -1 +1,3 @@
-JWT_SECRET='SECRET'
\ No newline at end of file
+JWT_SECRET='SECRET'
+NEXT_PUBLIC_RAPID_API_KEY='ac41cab2aamsh587d2717936bec9p185b22jsna20c36b1f5ef'
+NEXT_PUBLIC_FIREBASE_API_KEY='AIzaSyCiaPye-FbO69Lbh4algyKcOHGfG8IWsSg'
```

Comment:
I've got your secrets now 😈

Reply:
How do I undo this properly though?😬

- git rm --cached .env
- git commit -m "Git remove .env " ? would this be okay, or is there a more proper way?

Reply:
@ikulis-profico is a master of git :D

Reply:
You would need to do interactive rebase of the entire repo (main branch)
Steps:
- Find the commit where you first added the key to the repo (let's call it `KeyCommit`)
- Rebase interactive (git rebase -i ) to the commit before the `KeyCommit` and mark `KeyCommit` for edit (`e` flag)
- During edit mode, remove the key and use .env or something similar
- Preform rebase fixing conflicts that edit might have caused
- ???
- Profit!

You would have to force push main after 🥉 

Easier way would be to just invalidate the leaked key, remove it from the repo via a new commit and use .env for future commits. In this case if anyone finds your old key it would be unusable :) 
This is the prefered way of fixing this kind of issue but if you want to practice git-jitsu you can.
Make sure to `git tag` current main branch so you can easily `git reset --hard <tagname>` in case you mess up.

Reply:
It is a practice private repo so you don't need to do anything unless you have credit card linked to the key. If not, no problem ^ ^

Reply:
Thank you, I think I should be fine then with leaving it as it is, but I'll see about trying out your suggestion of how to remove it in the following days :)